one_crl_downloader.py

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+import os
+import sys
+from shutil import rmtree
+
+logger = logging.getLogger(__name__)
+
+
+class OneCRLDownloader(object):
+
```

Comment:
No, main.py is already too crammed and is is need of cleanup. Just remove the class definition and staticmethod declaration. See firefox_extractor.py.